src/bitcoin-witness-commitment.js

```diff
@@ -1,5 +1,5 @@
 import { BitcoinTransaction } from 'bitcoin-block'
-import { CID } from 'multiformats'
+import { create as createCID, asCID } from 'multiformats/cid'
```

Comment:
Nit: Diff may have end even smaller if you did

```suggestion
import * as CID from 'multiformats/cid'
```

Reply:
Fair, although there's so much hanging off `CID` now that I feel uncomfortable using it (a mental nit more than anything), including `CID.CID`.

But if you were to make an example usage of the new CID API, would you advise `* as CID` or the specific pieces? Let's make this an exemplar that we can point to until we have more & better docs.